src/components/02_atoms/OpsModalButton/OpsModalButton.js

```diff
@@ -23,6 +23,9 @@ function Transition(props) {
  * Custom Properties :-
  *   title, string to appear in the modal dialog.
  *   text, string to appear below the title.
+ *   cancelText, string to appear in the cancel button.
```

Comment:
I an wondering: Could we document proptypes like this? https://react-styleguidist.js.org/docs/documenting.html This seems to be a standard which we could show in styleguides at some point.

Reply:
Thanks for the feedback ... a good idea as usual, I was kinda making it up as I went along.  I have updated the PR. Also this is the first time we are using the "extends React.component" template ... I looked for other places in the code base to also modify.. but hey nothing doing